### DISCUSSION TITLE
Private Wapm repository for Wasm packages ?

### DISCUSSION THREAD

**Discussion by theagilecoder** [wasmerio/wasmer#3952] [Ideas] [answered]
### Summary

Hi, just like docker has a free public repository and paid private repository; does the Wapm package manager plan to have a private repository in the future ?

If an organizaton wants to host its wasm packages but doesnt want to expose it to public, private repository will be helpful.

**Comment by theduke** [comment, marked as answer]
Yes, this is definitely planned. 

In fact, you might get an update on this *very* soon.
